apps/emqx/etc/emqx.conf

```diff
@@ -80,12 +80,36 @@ broker {
   ## are mostly published to topics with large number of levels.
   ##
   ## NOTE: This is a cluster-wide configuration.
-  ## It rquires all nodes to be stopped before changing it.
+  ## It requires all nodes to be stopped before changing it.
   ##
   ## @doc broker.perf.trie_compaction
   ## ValueType: Boolean
   ## Default: true
   perf.trie_compaction = true
+
+
+  hot_config_loader {
```

Comment:
maybe `cluster_calls` ?

Reply:
If we move modules to emqx_machine, the cluster_call is great!